src/functions/chk_file.sh

```diff
@@ -81,10 +84,14 @@ chk_file_list() {
     if [[ "$(find "${CHK_FILE}" \( -perm -004000 -o -perm -002000 \) -type f -print)" ]]; then
       echo_message "\033[37;41m${CHK_FILE}\033[m\n" ",${CHK_FILE}\n" " filename='${CHK_FILE}'/>\n" " } }"
     else
-      echo_message "${CHK_FILE}\n" ",${CHK_FILE}\n" " filename='${CHK_FILE}'/>\n" " },\n"
+      LINE_ENDING=" },\n"
+      if [[ $CURRENT_LINE_NUMBER -eq $LAST_LINE_NUMBER ]]; then
+        LINE_ENDING=" }\n"
+      fi
+      echo_message "${CHK_FILE}\n" ",${CHK_FILE}\n" " filename='${CHK_FILE}'/>\n" "${LINE_ENDING}"
     fi
 
-  done < "${CHK_FILE_LIST}"
+  done <"${CHK_FILE_LIST}"
```

Comment:
I think shfmt is failing on this space removal

Reply:
https://github.com/mvdan/sh

Reply:
weird,  shfmt in vscode formatted by removing that space..

committed again
now the hooks passed locally ( thanks for the tips )